lib/src/Uncertainty/Distribution/openturns/GeneralizedExtremeValueFactory.hxx

```diff
@@ -60,25 +61,32 @@ public:
   GeneralizedExtremeValue buildAsGeneralizedExtremeValue(const Point & parameters) const;
   GeneralizedExtremeValue buildAsGeneralizedExtremeValue() const;
 
-  /** Maximum likelihood */
-  GeneralizedExtremeValue buildMethodOfLikelihoodMaximization(const Sample & sample) const;
-  DistributionFactoryLikelihoodResult buildMethodOfLikelihoodMaximizationEstimator(const Sample & sample) const;
-
   /** Profiled maximum likelihood */
   ProfileLikelihoodResult buildMethodOfProfileLikelihoodMaximizationEstimator(const Sample & sample) const;
   GeneralizedExtremeValue buildMethodOfProfileLikelihoodMaximization(const Sample & sample) const;
 
-  /** R largest order statistics */
-  DistributionFactoryResult buildRMaximaEstimator(const Sample & sample, const UnsignedInteger r = 0);
-  GeneralizedExtremeValue buildRMaxima(const Sample & sample, const UnsignedInteger r = 0);
-  UnsignedInteger buildBestRMaxima(const Sample & sample, const Indices & r, Point & logLikelihoodOut);
+  /** Maximum likelihood r largest order statistics */
+  DistributionFactoryLikelihoodResult buildMethodOfLikelihoodMaximizationEstimator(const Sample & sample, const UnsignedInteger r = 0) const;
+  GeneralizedExtremeValue buildMethodOfLikelihoodMaximization(const Sample & sample, const UnsignedInteger r = 0) const;
 
   /** Time-varying */
   TimeVaryingResult buildTimeVarying(const Sample & sample,
-                                     const Mesh & mesh,
+                                     const Sample & timeStamps,
                                      const BasisCollection & basisCollection,
-                                     const Function & inverseLinkFunction = Function()) const;
-
+                                     const Function & inverseLinkFunction = Function(),
+                                     const String & initializationMethod = ResourceMap::GetAsString("GeneralizedExtremeValueFactory-InitializationMethod"),
+                                     const String & normalizationMethod = ResourceMap::GetAsString("GeneralizedExtremeValueFactory-NormalizationMethod")) const;
```

Comment:
What about removing initializationMethod & normalizationMethod arguments ?
this function already has enough and these can be set in the ResourceMap anyways

Reply:
It was asked by our internal users, in the spirit of the R package ismev. The use of ResourceMap is not straightforward for most users.